### ISSUE TITLE
Some placeholders have no output + add rounded placeholders or output rounded values

### ISSUE THREAD

**Issue by RaptorXilef** [Iridium-Development/IridiumSkyblock#832]
**Describe the Bug and Feature Request**
1. There are several placeholders in IridiumSkyblock. Some of them have an output if you want to use them in other plugins. For example, if we insert them into our scoreboard plugin.  However, other placeholders work wonderfully. 

**Placeholder without output:**
- %iridiumskyblock_island_rank%
- %iridiumskyblock_island_experience_required%
- %iridiumskyblock_island_experience_remaining%
- %iridiumskyblock_island_overworld_biome%
- %iridiumskyblock_island_nether_biome%
- %iridiumskyblock_island_end_biome%
- %iridiumskyblock_island_upgrade_member_amount%
- %iridiumskyblock_island_upgrade_warp_amount%
- %iridiumskyblock_island_upgrade_size_dimensions%
- %iridiumskyblock_island_members%
- %iridiumskyblock_island_visitors%
- %iridiumskyblock_island_players%
- %iridiumskyblock_island_member_names%
- %iridiumskyblock_island_visitor_names%
- %iridiumskyblock_island_player_names%

**Working placeholders:**
- %iridiumskyblock_island_name%'
- %iridiumskyblock_island_owner%
- %iridiumskyblock_island_bank_money%
- %iridiumskyblock_island_bank_experience%
- %iridiumskyblock_island_bank_crystals%
- %iridiumskyblock_island_level%
- %iridiumskyblock_island_experience%
- %iridiumskyblock_island_value%

**Expected behavior**
An output of a string or a value.

<br><br><br>

2. If we use two specific placeholders, these are displayed with far too many decimal places, which has a negative impact on clarity. See screenshots.
- %iridiumskyblock_island_bank_money% -> is: 22018.916400000002       -> expected: 22018.91
- %iridiumskyblock_island_value% -> is: 272.40000000000003       -> expected: 272.4

The unexpected expenditure at %iridiumskyblock_island_bank_money% only occurs when a player deposits all his money into the bank. If only full amounts are deposited, the value is displayed correctly. I suspect that in this case Vault passes a value with more than 12 decimal places to IridiumSkyblock, so that it automatically continues to work with this value. Perhaps a solution could be found at this point. 
The unexpected output at %iridiumskyblock_island_value% always occurs.

**Expected behavior**
The output of a value rounded to max. 2 decimal places. 

**Wish**
Can you limit the output of these variables/placeholders to two decimal places or insert two additional placeholders that also output the rounded values?

**Server and Plugin Version**
`/version` 
```
Previous version: git-Paper-461 (MC: 1.20.4)
```

`/is about`
```
- Plugin Name: IridiumSkyblock
- Plugin Version: 4.0.9
- Plugin Author: Peaches_MLG
```

**Screenshots**
![2024,04,05-18,58,47-001](https://github.com/Iridium-Development/IridiumSkyblock/assets/31962607/fb59e86e-25a2-45e5-b088-6c4d34000843) __________ ![2024,04,05-19,01,08-001](https://github.com/Iridium-Development/IridiumSkyblock/assets/31962607/ee9d9341-7552-4df1-a6b2-beec815206e9)

Should you require further information from me, please do not hesitate to contact me at any time.

**Comment by sh0inx** [comment]
does this also happen when setting the number formatter in settings?

``configuration.yml``
```yaml
numberFormatter:
  numberAbbreviationDecimalPlaces: 2
  ```

**Comment by RaptorXilef** [comment]
Yes, I have set the settings as you described. The BUG described above in the display of decimal places still exists.
Even changing the value to 1 or 0 does not change anything.
My current attitude is:

`configuration.yml`
```
numberFormatter:
  numberAbbreviationDecimalPlaces: 2
  thousandAbbreviation: "K"
  millionAbbreviation: "M"
  billionAbbreviation: "B"
  trillionAbbreviation: "T"
  displayNumberAbbreviations: true
```

**Comment by sh0inx** [comment]
potentially fixed in #896

**Comment by glenngenre** [comment]
Closed since #896 is merged.
